setup_monitor.py

```
@@ -0,0 +1,193 @@
+            print(f"❌ Error reading config: {str(e)}")
+    else:
+        print("❌ Configuration file not found")
+    
+    # Check cron job
+    try:
+        result = subprocess.run(['crontab', '-l'], capture_output=True, text=True)
+        if result.returncode == 0 and "flight_monitor.py" in result.stdout:
+            print("✅ Cron job is active")
+        else:
+            print("❌ No cron job found")
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)